### DISCUSSION TITLE
MUX/DEMUX Suggestions & Issues

### DISCUSSION THREAD

**Discussion by Jade-Harleyy** [FakeFishGames/Barotrauma#14259] [Unstable feedback]
### Suggestions
1. Merge the MUX/DEMUX components into 1 single component (DE/MUX or I/O Selector Component).

### Issues
1. Pins on the DEMUX component's sprite are on the wrong side.
2. Nether component fits perfectly into the grid like every other component.
3. Nether component can be mirrored on the X axis. (Seems to be an issue with every component now.)

**Comment by ven1883** [comment]
I really quite like these, with my only complaint being that you can not split or combine signals with them.
I would really like some signal combiner and splitters that aren't sketchy conc and regex components stuck together.
